dev-tools/go.mod

```diff
@@ -0,0 +1,49 @@
+module github.com/elastic/fleet-server/dev-tools
+
+go 1.24.0
+
+tool (
+	github.com/deepmap/oapi-codegen/v2/cmd/oapi-codegen
+	github.com/elastic/go-json-schema-generate/cmd/schema-generate
+	github.com/elastic/go-licenser
+	github.com/jstemmer/go-junit-report
+	go.elastic.co/go-licence-detector
+	golang.org/x/perf/cmd/benchstat
+)
+
+require github.com/elastic/elastic-agent-libs v0.18.12
```

Comment:
This is listed as a requirement because of the `dev-tools/e2e/validatecerts.go` script that is used